static/js/task/controllers/task.controller.js

```diff
@@ -74,23 +74,15 @@
 
         function skip() {
             Task.skipTask(self.task_id).then(function success(data, status) {
-                    if (data[1]==200){
```

Comment:
in what way did it break it?
well you could go about status or by data with a message.
it's a SUCCESS status as the task is skipped successfully and if there is no other left you get 204 else 200